### PR TITLE
Merging styling with develop

### PR DESCRIPTION
**Pull request by DXCanas**
Needed to resolve some merge conflicts, so used my local copy instead of merging the 2 branches.

### CODE REVIEW

**Comment by christianmemije** [comment]
weird

image

**Comment by DXCanas** [comment]
re: @christianmemije are these deal breakers - @indirectlylit ? @jtamiace 

This almost runs into the same issue I'm getting in #91; luckily `$core-action-light` is only used in k-navbar. That was carryover from the ported styles.

I think we need to think about our theming practices in the near future.

File: kolibri_instant_schools_plugin/assets/src/styles/theme.styl [hunk 1]
```diff
@@ -24,7 +23,7 @@ $app-bar-title-icon = url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAA
 */
 
 $core-font = 'NotoSans', 'sans-serif'
-
+$core-action-light = #E2D1E0
```
Comment (DXCanas) [inline kolibri_instant_schools_plugin/assets/src/styles/theme.styl]: image
@christianmemije

Comment (DXCanas) [reply kolibri_instant_schools_plugin/assets/src/styles/theme.styl]: this is being set to the value in [Kolibri's theme file](kolibri/core/assets/src/styles/core-theme.styl). Every variable defined there has to be defined here.

**Comment by christianmemije** [comment]
@DXCanas We still need to override the logo component to use the VF logo.

File: kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
     <ui-toolbar type="colored" textColor="white">
       <template slot="icon">
-        <ui-icon class="app-bar-icon"><logo/></ui-icon>
+        <img src="../img/instant-school-logo.png" class="app-bar-icon">
```
Comment (christianmemije) [inline kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue]: Also, let's just use the logo component here too?

Comment (DXCanas) [reply kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue]: Per discussion, ignoring this comment

**Comment by DXCanas** [comment]
@christianmemije done

File: kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue [hunk 3]
```diff
@@ -368,48 +356,11 @@
     margin-right: auto
     width: ($iphone-5-width - 20)px
 
-  .terms
```
Comment (DXCanas) [inline kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue]: :cry:

**Pull request merged**